packages/babel-plugin-open-source/babel.js

```diff
@@ -16,7 +16,8 @@ module.exports = declare((api) => {
       enter: (_, state) => {
         state.file.set('hasJSX', false);
 
-        let editor = state.opts && state.opts.editor ? state.opts.editor.toLowerCase() : 'vscode';
+        const editorInOpts = state.opts && state.opts.editor ? state.opts.editor.toLowerCase() : 'vscode'
+        let editor = process.env.NODE_ENV === 'production' ? 'github' : editorInOpts;
```

Comment:
maybe this should be after editorInENV as well? 🤔

Reply:
The idea behind here was that:
1 - Users would very likely not set the editor as an environment variable in their CI, so. it will very likely yield undefined
2 - When testing building their app in prod mode and testing it locally, the editor from the .env file will be respected